services/QuillLMS/app/services/application_service.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class ApplicationService
-  def self.run(*args, **options, &block)
-    new(*args, **options, &block).run
+  def self.run(...)
+    new(...).run
   end
 end
```

Comment:
Note the `...` for arguments [forwarding](https://blog.saeloun.com/2019/12/04/ruby-2-7-adds-new-operator-for-arguments-forwarding/)